cypress/integration/augmintbasic.js

```diff
@@ -20,13 +20,15 @@ describe("Augmint basic e2e", function() {
             .should("contain", "Selected: loan product " + (prodId + 1));
         // NB: only works with integers, see: https://github.com/cypress-io/cypress/issues/1171
         cy
-            .get("#disbursedTokenAmount")
+            .get("[testid='loanTokenAmountField']")
```

Comment:
I'd rather go with a custom data attribute in the long run, like `data-test-id="loanTokenAmountFiled"`

Reply:
this is change to a data attribute. or do i miss something?
(it's in the trello backlog to change everywhere else in the tests too)

Reply:
left the same comment on the [trello card](https://trello.com/c/2sIyNGw6/20-replace-id-s-with-a-custom-testid-tag#comment-5a91dcc032d42eed7974ac3b). It doesn't block the merge.

Reply:
ah, ok you mean use `data-test-id` instead of `testid`? 
unfortunately I had some issues with dashes in the custom tag name, can't recall where ... we can give one more try :)